spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/GRU.scala

```diff
@@ -33,9 +34,16 @@ import scala.reflect.ClassTag
 @SerialVersionUID(6717988395573528459L)
 class GRU[T : ClassTag] (
   val inputSize: Int,
-  val outputSize: Int)
+  val outputSize: Int,
+  val wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out?